test_regress/t/t_randcase_bad.out

```diff
@@ -1,2 +1,2 @@
-[0] %Error: t_randcase_bad.v:12: Assertion failed in top.t.unnamedblk2_1: All randcase items had 0 weights (IEEE 1800-2023 18.16)
+All randcase items had 0 weights (IEEE 1800-2023 18.16)
```

Comment:
This error change doesn't seem to be right, lost line number.

Reply:
Fixed